### DISCUSSION TITLE
Error:Cannot read properties of undefined (reading 'split')

### DISCUSSION THREAD

**Discussion by glzat** [community/community#103065] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When using Github Copilot, the error "Cannot read properties of undefined (reading 'split')" is reported, which only occurs in the second and subsequent commands of a new dialog
![image](https://github.com/community/community/assets/104402331/291527b1-f237-42eb-86a6-91b1c2dfe79d)

**Comment by andybrandt** [comment]
Have same problem:
![image](https://github.com/community/community/assets/5551980/7792dd4f-7fb2-44c1-a0a1-fe6d57ec805e)

**Reply by glzat** [reply]
OK bro.I had solved this problem.Just delete "Github Copilot" and install it again.Then it can work.
Just like this:(I am sorry that I'm not accustomed to using English. )
![image](https://github.com/community/community/assets/104402331/8c42c6aa-f486-4efc-baae-6b24cfb08252)
